components/nexus-mime/src/main/resources/builtin-mimetypes.properties

```diff
@@ -30,3 +30,4 @@ tgz = application/x-tgz,application/x-gzip,application/gzip
 bz2 = application/x-bzip2
 tbz = application/x-bzip2,application/x-bzip
 nupkg = application/zip
+kar = application/zip,application/x-zip-compressed
```

Comment:
No reason, I just went the "safe side" and added all the Tike mime magic detection detected (as seen on DEBUG level of repository logger). But basically it's the leftmost that is considered the "typical" type from this properties file.